src/utils/getDefaultDateValue.ts

```diff
@@ -1,8 +1,8 @@
-export const getDefaultDateValue = (currentDate) => {
+export const getDefaultDateValue = (currentDate?: Date): string => {
   const date = currentDate ?? new Date();
 
-  let day = date.getDate();
-  let month = date.getMonth() + 1;
+  let day: number | string = date.getDate();
+  let month: number | string = date.getMonth() + 1;
```

Comment:
This should be `number`?   also we can also make use of ts inference and not specify any type

Reply:
Few lines below this number is casted to a string via `template literals` and the function is returning a string. Refactoring -> will add parsedMonth and day in separate variables and leave this to be a number by ts inference! 👍